packages/embark/src/lib/utils/solidity/remapImports.ts

```diff
@@ -69,11 +69,12 @@ const buildNewFile = (file: File, importPath: string) => {
     return new File({ externalUrl: importPath, type: Types.http });
   }
 
+  importPath = path.normalize(importPath);
+
   // imported from node_modules, ie import "@aragon/os/contracts/acl/ACL.sol"
-  if (isNodeModule(importPath)) {
+  if (isUnresolvedNodeModule(importPath)) {
     from = resolve(importPath);
-    to = importPath.includes(fs.dappPath(".embark")) ? importPath : fs.dappPath(".embark", "node_modules", importPath);
-    to = path.normalize(to);
+    to = path.normalize(importPath.includes(fs.dappPath(".embark")) ? importPath : fs.dappPath(".embark", "node_modules", importPath));
```

Comment:
Doesn't `dappPath` already normalize? Also, we normalized `importPath` above, so I'm pretty sure we can remove `path.normalize` here.

Reply:
Good catch, making the change now.